Packs/OktaITAdmin/Integrations/Okta_IT_Admin/Okta_IT_Admin.yml

```diff
@@ -0,0 +1,359 @@
+commonfields:
+  id: Okta IT Admin
+  version: -1
+name: Okta IT Admin
+display: Okta IT Admin
+category: IT Services
+description: Integration with Okta's cloud-based identity management service
+configuration:
+- display: Okta URL (https://<domain>.okta.com)
+  name: url
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: API Token (see Detailed Instructions)
+  name: apitoken
+  defaultvalue: ""
+  type: 4
+  required: true
+- display: Fetch incidents
+  name: isFetch
+  type: 8
+  required: false
+- display: Incident type
+  name: incidentType
+  type: 13
+  required: false
+- display: Custom Mapping for Create User
+  name: customMappingCreateUser
+  defaultvalue: ""
+  type: 12
+  required: false
+  additionalinfo: An optional custom mapping that takes custom values in the SCIM
+    data into the integration.
+- display: Custom Mapping used for Update User
+  name: customMappingUpdateUser
+  defaultvalue: ""
+  type: 12
+  required: false
+  additionalinfo: An optional custom mapping that takes custom values in the SCIM
+    data into the integration.
+- display: ""
+  name: fetchLogsQuery
+  defaultvalue: ""
+  type: 12
+  required: false
+- display: ""
```

Comment:
please add display (description) for these parameters

Reply:
Done